speclang_modern.tex

```diff
@@ -168,6 +168,8 @@ \section{Logic expressions}
   $\{$\lstinline|<|, \lstinline|<=|, \lstinline|==|$\}$ or to
   $\{$\lstinline|>|,\lstinline|>=|,\lstinline|==|$\}$. Expressions such as
   \lstinline|x < y > z| or \lstinline|x != y != z| are not allowed.
+  Note that consecutive comparison operators are allowed only 
```

Comment:
I could imagine so, but the grammar only allows consecutive operations in a predicate (Figures 2.1 and 2.2).
Consequently in a predicate position, a < b < c ? P : Q  is ambiguous, because it is not known whether a<b<c is a consecutive operation predicate or is (a<b)<c, a likely unintended term.

(For me, I would dispense with the distinction between predicates and terms altogether, as in SMT-LIB and elsewhere, but that would be a big change for Frama-C.)